### PR TITLE
Fix disconnect timeout was not working 

### PR DESCRIPTION
**Pull request by metalefty**
Timeout value wasn't read from environment variable.
And some logs improvements.

### CODE REVIEW

File: module/rdpClientCon.c [hunk 1]
```diff
@@ -1197,6 +1200,40 @@ rdpClientConInit(rdpPtr dev)
         g_sck_listen(dev->listen_sck);
         rdpClientConAddEnabledDevice(dev->pScreen, dev->listen_sck);
     }
+
+    ptext = getenv("XRDP_SESMAN_MAX_IDLE_TIME");
+    if (ptext != 0)
+    {
+    }
+    ptext = getenv("XRDP_SESMAN_MAX_DISC_TIME");
+	if (ptext != 0)
```
Comment (speidy) [inline module/rdpClientCon.c]: Please fix indentation

File: module/rdpClientCon.c [hunk 2]
```diff
@@ -1197,6 +1200,40 @@ rdpClientConInit(rdpPtr dev)
...
+        }
+    }
+
+    if (dev->do_kill_disconnected && (dev->disconnect_timeout_s < 60))
+    {
+        dev->disconnect_timeout_s = 60;
+    }
+
+    LLOGLN(0, ("rdpClientConInit: kill disconnected [%d] timeout [%d] sec\n",
+           dev->do_kill_disconnected, dev->disconnect_timeout_s));
```
Comment (speidy) [inline module/rdpClientCon.c]: Fix indent

**Comment by speidy** [comment]
+1

**Comment by metalefty** [comment]
Fixed indentation and force pushed.

I tested xorgxrdp properly exits after disconnect. Here's the summary of log.

```
[  3203.147] rdpClientConInit: kill disconnected [1] timeout [120] sec
(snip)
[  3238.586] rdpClientConRecv: g_sck_recv failed(returned 0)
[  3238.586] rdpClientConDisconnect:
[  3238.586] rdpClientConDisconnect: starting g_dis_timer, exit after 120 seconds
[  3238.586] rdpClientConDisconnect: clientCon removed from dev list
[  3238.586] rdpClientConRecvMsg: error
[  3238.586] rdpClientConCheck: rdpClientConGotData failed
[  3358.656] rdpDeferredDisconnectCallback: disconnect timeout exceeded, exiting
[  3358.662] rdpmouseControl: what 2
[  3358.662] rdpmouseDeviceOff:
[  3358.662] rdpkeybControl: what 2
[  3358.662] rdpkeybDeviceOff:
[  3358.663] rdpkeybControl: what 3
[  3358.663] rdpkeybUnInit: drv 0x7f379e496700 info 0x7f379e7055b0, flags 0x0
[  3358.663] rdpUnregisterInputCallback: proc 0x7f3794e28450
[  3358.663] rdpmouseControl: what 3
[  3358.663] rdpmouseUnInit: drv 0x7f379e495ff0 info 0x7f379e57bc10, flags 0x0
[  3358.663] rdpUnregisterInputCallback: proc 0x7f379502bcb0
[  3358.663] rdpCloseScreen:
[  3358.663] xorgxrdpDownDown:
[  3358.663] xorgxrdpDownDown: 1
[  3358.663] rdpClientConDeinit:
[  3358.663] rdpClientConDeinit: deleting file /tmp/.xrdp/xrdp_display_12
[  3358.663] (II) Server terminated successfully (0). Closing log file.
```

File: module/rdpClientCon.c [hunk 3]
```diff
@@ -1197,6 +1200,40 @@ rdpClientConInit(rdpPtr dev)
         g_sck_listen(dev->listen_sck);
         rdpClientConAddEnabledDevice(dev->pScreen, dev->listen_sck);
     }
+
+    ptext = getenv("XRDP_SESMAN_MAX_IDLE_TIME");
```
Comment (proski) [inline module/rdpClientCon.c]: I would prefer parsing for XRDP_SESMAN_MAX_IDLE_TIME to be added when it's actually supported. It's disappointing to find such code when figuring out why a feature doesn't work.

Comment (metalefty) [inline module/rdpClientCon.c]: Indeed.

**Comment by moobyfr** [comment]
works fine

**Comment by metalefty** [comment]
Thanks for testing. I think it's ready to merge.

**Comment by jsorg71** [comment]
merged as b2e1baf8cb34dd09e9f7180833ea543eb26bc32c

**Pull request closed without merging**